packages/ai/core/generate-text/generate-text.ts

```diff
@@ -573,26 +573,22 @@ A function that attempts to repair a tool call that failed to parse.
         }),
       );
 
+      const resolvedOutput = await output?.parseOutput(
```

Comment:
Instead of passing an `outputResolver()` callback, the output needs to be computed ahead of instantiating the `DefaultGenerateTextResult` class, in order to retain the synchronous `.experimental_output` getter API